src/main.cc

```diff
@@ -53,10 +53,7 @@ bpo::variables_map parse_options(int argc, char *argv[]) {
       ("loglevel", bpo::value<int>(), "set log level 0-4 (trace, debug, warning, info, error)")
       ("config,c", bpo::value<std::string>()->required(), "toml configuration file")
       ("gateway-http", bpo::value<bool>(), "enable the http gateway")
-      ("gateway-rvi", bpo::value<bool>(), "enable the rvi gateway")
       ("gateway-socket", bpo::value<bool>(), "enable the socket gateway")
-      ("gateway-dbus", bpo::value<bool>(), "enable the D-Bus gateway")
-      ("dbus-system-bus", "Use the D-Bus system bus (rather than the session bus)")
```

Comment:
@cajun-rat @OYTIS do we know of anyone that might be using these options, even if it is to just set them to `false`? I think we can largely not worry about that, but I'm just trying to be careful.

Reply:
Good question.  I think your concern is valid.  For people who are using these features, exploding violently is probably the best option--they should find out they they can't do stuff as soon as possible.

The case that you were thinking of though was where people had forced that value to 'off' (the default).  It would be nice to not break those guys.  The simplest solution I can think of is to document these options as 'deprecated' then check then and throw an error if they are set to true.  The cost would be that we would have to maintain that extra code in the future.

Given the balance between a mild annoyance to somewhere between 0 and 10 users and our long term maintenance costs, I think just dropping the options is fine.

Reply:
Agreed. I think the chance that someone set one of these options explicitly to 'off' is unlikely, and the fix is simple anyway.